### PR TITLE
App Lab: Allow a wider range of values in rgb block

### PR DESCRIPTION
**Pull request by caleybrock**
Numbers can be negative, positive, and decimal values.

### CODE REVIEW

**Comment by caleybrock** [comment]
originally unsure about the spec here, but it's @poorvasingal and baker approved.

File: apps/src/applab/commands.js [hunk 1]
```diff
@@ -539,10 +539,15 @@ applabCommands.setStrokeWidth = function (opts) {
...
-  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.b, 'number', 0, 255);
+  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.r, 'number');
+  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.g, 'number');
+  apiValidateTypeAndRange(opts, 'rgb', 'number', opts.b, 'number');
   apiValidateTypeAndRange(opts, 'rgb', 'number', opts.a, 'number', 0, 1, OPTIONAL);
+
+  // Convert any decimal values into integers between 0 and 255
+  opts.r = Math.min(255, Math.max(0, parseInt(opts.r)));
+  opts.g = Math.min(255, Math.max(0, parseInt(opts.g)));
+  opts.b = Math.min(255, Math.max(0, parseInt(opts.b)));
```
Comment (Bjvanminnen) [inline apps/src/applab/commands.js]: Might want to specify a radix for parseInt (i.e. `parseInt(opts.r, 10)`). I'm a little surprised our linter doesn't look for this. See http://eslint.org/docs/rules/radix for why (or run `parseInt(017)` and see if the result is what you expect.

Comment (Bjvanminnen) [reply apps/src/applab/commands.js]: Do we have any tests for this method?

**Comment by caleybrock** [comment]
Realized I didn't actually need parseInt, just to round it since I already validate it's a number. Also added some test cases.

**Pull request merged**